### PR TITLE
Refactor loginRequired.js with Aphrodite and commonForm

### LINKED ISSUE #8009: Refactor loginRequired.js with Aphrodite and commonForm
## Test plan

https://github.com/brave/browser-laptop/pull/8011#issue-218586228

----

**Describe the issue you encountered:** Refactor loginRequired.js with Aphrodite, applying `.commonForm` (which shall be implemented with PR #7995) instead of `.genericForm`

Currently there are only three other instances where `.genericForm` is applied (autofillAddressPanel.js, autofillCreditCardPanel.js, and addEditBookmarkHanger.js), and in every case styles which genericForm defines are overwritten by what commonForm is going to do so (title color, padding inside the dialog, etc). Keeping both of them is a cause of style inconsistency.

- Any related issues: #7990 #7996 #8010

### PR DESCRIPTION
**Pull request by luixxiul**
Refactor loginRequired.js with Aphrodite and commonForm

- textbox and textbox__outlineable were copied from textbox.js to commonStyles.js

Since FormTextbox cannot be used for the input elements (because the prop `ref={}` cannot be got), I copied the styles applied for that element and applied to them. See: #7164 (comment)

The labels and input forms were grouped and placed with display:flex and justify-content:space-between. Also the elements inside each wrapper were aligned equally to make the length of the input forms always equal (l10n friendly).

Also colons in the label were removed to make the style consistent.

Closes #8009
Addresses #8010

Auditors:

Test Plan:
1. Visit http://browserspy.dk/password.php
2. Click "password-ok.php" link
3. Make sure you can log in successfully with the given credential
4. Change the lang setting on about:preferences
5. Try the same steps above and make sure the length of the input forms is equal

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [ ] Ran `git rebase -i` to squash commits (if needed).

### CODE REVIEW

**Comment by luixxiul** [comment]
From:
screenshot 2017-04-01 2 48 10

To:
screenshot 2017-04-01 3 35 25

**Comment by luixxiul** [comment]
@NejcZdovc would you mind reviewing the PR when you have some time? There are several PRs dependent on this one.

As I will push the PR once again to the master once everything which should include `commonForm` is refactored (to avoid style inconsistency over the releases), you would not need to be so strict. Thanks!

**Comment by NejcZdovc** [comment]
@luixxiul sure, will review it in couple of hours

File: js/components/loginRequired.js [hunk 1]
```diff
@@ -73,31 +85,66 @@ class LoginRequired extends React.Component {
...
+              <label className={css(commonFormStyles.input__bottomRow)} data-l10n-id='basicAuthPasswordLabel' htmlFor='loginPassword' />
             </div>
-            : null
-          }
-          <div className='formRow'>
-            <Button l10nId='cancel' className='whiteButton' onClick={this.onClose} />
-            <Button l10nId='ok' className='primaryButton' onClick={this.onSave.bind(this)} />
+            {
+              !this.isFolder
+              ? <div id='loginInput' className={css(
```
Comment (NejcZdovc) [inline js/components/loginRequired.js]: Do we need this ID?

**Comment by bsclifton** [comment]
This looks great! Awesome job, @luixxiul! すごい!!! 😄

**Comment by luixxiul** [comment]
@bsclifton Arigato!!

File: js/components/loginRequired.js (hunk 1 above)
Comment (luixxiul) [inline js/components/loginRequired.js]: Not quite sure. We could remove them later if we would.

**Comment by luixxiul** [comment]
Note: this change is not yet available on master. I will push a PR to the master once every change which `commonForm` requires are made on `refactoring-aphrodite` (to avoid style inconsistency over the releases).

**Pull request merged**